homepage/locales/vn.ts

```diff
@@ -0,0 +1,83 @@
+export default {
+  translation: {
+    common: {
+      openInBrowser: 'Mở trong trình duyệt',
+      downloadApp: 'Tải ứng dụng',
+      fileSystemBasedStorage: 'Lưu trữ trên file',
+      comingSoon: 'Sắp ra mắt',
+      webApp: 'Ứng dụng web',
+      desktopApp: 'Ứng dụng desktop',
+      mobileApp: 'Ứng dụng điện thoại',
+    },
+    header: {
+      forTeams: 'Dành cho teams',
+    },
+    hero: {
+      title: 'Tăng sự hài lòng, năng suất và sáng tạo.',
+      subtitle:
+        "Một ứng dụng ghi chú markdown hiệu quả cho lập trình viên và hoàn toàn open-source.",
+    },
+    boostHub: {
+      title: 'Boost Note dành cho teams',
+      description1:
+        'Chúng tôi sẽ cho ra mắt 1 ứng dụng wiki đơn giản dành riêng cho lập trình viên, gọi là "BoostHub" vào tháng 6 năm 2020!',
+      description2:
+        'BoostHub hướng đến việc trở thành 1 công cụ wiki cho phép các team tương tác với nhau. Công cụ này sẽ giúp các teams tập trung mọi dữ liệu và cung cấp khả năng tìm kiếm trên dữ liệu của cả bên thứ nhất và bên thứ ba.',
+    },
+    features: {
+      title: 'Các chức năng',
+      cloudStorage: 'Lưu trữ đám mây',
+      cloudStorageDescription:
+        'Mọi ghi chú sẽ được lưu trữ an toàn trên đám mây và có thể truy cập trên mọi thiết bị.',
+      multiplePlatforms: 'Đa nền tảng',
+      multiplePlatformsDescription:
+        'Ứng dụng Boost Note có thể sự dụng dưới dạng: ứng dụng web, ứng dụng desktop và ứng dụng điện thoại.',
+      syntaxHighlight: 'Tô sáng cú pháp',
```

Comment:
Actually I would prefer to keep it as `Syntax Highlight` though
```suggestion
      syntaxHighlight: 'Nổi bật cú pháp',
```

Reply:
Let's just keep `Syntax Highlight` Vietnamese sounds awkward :rofl: